### ISSUE TITLE
Step 2  Creating as Apptainer Image on MRS UAV System

### ISSUE THREAD

**Issue by ZarrarHaider456** [ctu-mrs/mrs_singularity#18]
Hi,
    I am unable to execute commands and create the MRS UAV System. I have ubuntu 22.04 and I have app container installed. Kindly help me in this regard. I am coming to summer schools and need to get my system ready. Kindly guide me

**Comment by klaxalk** [comment]
Hey, which commands are you unable to execute? What output do they provide?

You don't need to create the image yourself now. You will get a pre-built image from us when the summer school starts. Right now, you can try to install the Apptainer. We provide installation script for Ubuntu (https://github.com/ctu-mrs/mrs_apptainer/blob/master/install/install_apptainer.sh).

**Comment by ZarrarHaider456** [comment]
Hi,
  I have app container installed on my system but I am unable to make Image of MRS UAV system Step 2? I have Ubuntu 22.04 and ROS 2 Kindly guide me would I be able to the packages which you will teach in the summer school?

**Comment by klaxalk** [comment]
Yes, it will probably work and if not, we will help you in person. However, to help you now, I really need to know what the problem is. What error does the script show?

**Comment by AndrasZabo** [comment]
Hey,
I have issues with this step 2 as well. 
I'm using ubuntu 22.04 as well, and here are the details, but please tell me if anything else is needed:

1.  [here](https://github.com/ctu-mrs/mrs_apptainer?tab=readme-ov-file#quick-start-guide-linux) step 1 is done
2. then at step 2 I run [this](https://github.com/ctu-mrs/mrs_apptainer/blob/master/recipes/stable_from_docker/build.sh) build.sh script: .../SummerSchools/MRS_2024/MRS_system$ bash build.sh
3. and receive the following error: .../SummerSchools/MRS_2024/MRS_system/recipe.def: no such file or directory
4. after that I downloaded the recipe.def file from [here](https://github.com/ctu-mrs/mrs_apptainer/blob/master/recipes/stable_from_docker/recipe.def) into the .../MRS_system directory, and then bash build.sh command executes
5. but at the end I get the following: 
.../SummerSchools/MRS_2024/MRS_system$ bash build.sh
INFO:    Starting build...
(...)
INFO:    Adding environment to container
INFO:    Adding runscript
INFO:    Creating SIF file...
FATAL:   While performing build: while creating SIF: while creating container: open .../SummerSchools/images/mrs_uav_system.sif: no such file or directory 

Could you please tell me what am I making wrong, or what may be the issue? Thank you!
